Dashboard/citymanage/static/css/dashboard.css

```diff
@@ -0,0 +1,109 @@
+
+html, body {
+    height: 100%;
+}
+
+.col-sm-3.sidenav.hidden-xs {
```

Comment:
col-sm-3, hidden-xs are not expressive. If they are from external library, it's fine.